sdks/python/setup.py

```diff
@@ -272,6 +162,8 @@ def get_portability_package_data():
   # structure must exist before the call to setuptools.find_packages()
   # executes below.
   generate_protos_first()
+  # We can't move the dependencies out of the setup call, otherwise
```

Comment:
Rather than phrase this as a negative, I would say "Keep all dependencies inlined in the setup call, otherwise..."

Reply:
SGTM - updated